table/tables/index.go

```diff
@@ -108,6 +108,19 @@ func (c *index) Create(sctx sessionctx.Context, txn kv.Transaction, indexedValue
 		return nil, err
 	}
 
+	var (
+		tempKey        []byte
+		keyVer         byte
+		keyIsRewritten bool
+	)
+	if !opt.FromBackFill {
```

Comment:
Where we set this option, I only see https://github.com/pingcap/tidb/pull/37709/files#diff-ec419f15a610ebac64eb260933636782073ec5722528c62cc646a9e73743bac8R60.

Reply:
Good catch.